src/resources/ha-style.ts

```diff
@@ -30,6 +30,7 @@ documentContainer.innerHTML = `<custom-style>
       --light-primary-color: #b3e5fC;
       --accent-color: #ff9800;
       --divider-color: rgba(0, 0, 0, .12);
+      --hyperlink-color: var(--primary-color);
```

Comment:
I am not opposed, but I also don't want to make rush decisions on adding new theming variables, since we have to support his forever. I've asked for some more opinions in the frontend dev channel.

Reply:
sure thing